__tests__/app.test.js

```diff
@@ -139,6 +139,42 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  it("?sort_by= query. Status 200: responds with articles sorted according to a valid column query (defaults to descending order)", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", {
+          descending: true,
+        });
+      });
+  });
+  it("?sort_by=&order= query. Status 200: responds with articles sorted by any valid column query and a specified order (asc)", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", {
+          ascending: true,
```

Comment:
There is no ascending option in jestSorted options. This does work though because if you don't have { descending:true } it defaults ascending